test/components/patient/patient_test.js

```diff
@@ -0,0 +1,12 @@
+import { renderComponent , expect } from '../../test_helper';
```

Comment:
I would just delete this test as this component does not exist. You can add a container test though (see `test/containers/patients_test.js` for ideas).